ci/jenkins/pipelines/skuba-conformance.Jenkinsfile

```diff
@@ -10,6 +10,23 @@ pipeline {
     }
 
     stages {
+
+        stage('Git Clone') { steps {
+            deleteDir()
+            checkout([$class: 'GitSCM',
+                      branches: [[name: "*/${BRANCH_NAME}"],[name: '*/master']],
```

Comment:
Shouldn't this match the name of the parameter in the job, which is `BRANCH`?

https://github.com/SUSE/skuba/blob/master/ci/jenkins/templates/conformance-template.yaml#L15-L17

Reply:
Im not sure, its set like this on other jobs and it seems to come from jenkins itself.

Let me look into it. I probably need to deploy a local jenkins to test this :/

Reply:
yup, you were right, updated!